ai/llmbot.py

```diff
@@ -221,6 +221,45 @@ def answer_webcontent(self, message_text: str, response_content: str) -> str | N
             logging.exception(e)
         return None
 
+    def generate_feedback_message(self, message_type: str) -> str:
```

Comment:
@Codegen-sh, could you include tests for `generate_feedback_message` function? You'll find the `llmbot` module tests in the `test_llmbot.py` file